.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Report a security vulnerability
-    url: https://loopback.io/doc/en/contrib/Reporting-issues.html#security-issues
-    about: Do not report security vulnerabilities using GitHub issues. Please send an email to `reachsl@us.ibm.com` instead.
```

Comment:
AFAIK, keeping this (somewhat redundant, but different) security contact link was on purpose; (see @bajtos' https://github.com/strongloop/loopback-next/pull/4783#discussion_r388282921).